apps/core/src/hooks/useGetRollingAverageApys.ts

```diff
@@ -60,12 +61,29 @@ export function useGetRollingAverageApys(numberOfValidators: number | null) {
         order: 'descending',
     });
 
+    const { data, isLoading } = useGetSystemObject();
+
     const apyByValidator =
         useMemo<ApyByValidator | null>(() => {
-            if (!validatorEpochEvents?.data) {
+            if (!validatorEpochEvents?.data || !data) {
                 return null;
             }
+            const { stakeSubsidyStartEpoch, epoch, activeValidators } =
+                data || {};
+            // return 0 for all validators if current epoch is less than the stake subsidy start epoch
+            if (epoch < stakeSubsidyStartEpoch) {
+                return activeValidators.reduce((acc, validator) => {
+                    acc[validator.suiAddress] = 0;
+                    return acc;
+                }, {} as ApyByValidator);
+            }
+
+            // The rolling average epoch is the current epoch - the stake subsidy start epoch
+            const avgEpochNumberAfterSubsidy =
```

Comment:
I'm a little confused why you're subtracting from the rolling average const, seems not quite right. seems like this should take the current epoch number into account otherwise this is just a constant itself

Reply:
This is done.